<launch>
    <node name="hsv_filter_node" pkg="vision" type="hsv_filter_node" output="screen">
        <param name="h_low" value="0" type="int" />
        <param name="s_low" value="0" type="int" />
        <param name="v_low" value="0" type="int" />
        <param name="h_high" value="180" type="int" />
        <param name="s_high" value="253" type="int" />
        <param name="v_high" value="103" type="int" />
        <remap from="/camera/image_raw" to="/uwsim/camera2" />
    </node>

    <node name="gate_detect_node" pkg="gate_detect" type="gate_detect" output="screen">
        <param name="cannyLow" value="50" type="int" />
        <param name="houghLinesThreshold" value="100" type="int" />
        <param name="houghLinesMinLength" value="125" type="int" />
        <param name="houghLinesMaxLineGap" value="150" type="int" />
        <param name="poleMax" value="50" type="int" />
        <param name="interpolationConstant1" value="81.88" type="double" />
        <param name="interpolationConstant2" value="-.92791" type="double" />
        <param name="lowVertThresh" value="150" type="int" />
        <param name="lowHorThresh" value="150" type="int" />
        <param name="width" value="500" type="int" />
        <param name="height" value="360" type="int" />
    </node>

</launch>
